### PR TITLE
Nerfs Sigils of Transgression/Submission

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Sigils of Transgression will now only stun for about 5 seconds, from around 10 seconds.
tweak: Sigils of Submission take about 5 seconds to convert a target, from around 3 seconds, and will glow faintly.
rscadd: Sigils of any type can be attacked with an open hand to destroy them, instead of requiring harm intent. Servants still require harm intent to do so.
/:cl:

Sigil conversion is still 100%, but there's a slightly longer interval where you can mess with the person on it, and sigils are easier to remove now.

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_scripture.dm [hunk 1]
```diff
@@ -317,14 +317,14 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 /datum/clockwork_scripture/create_object/sigil_of_submission //Sigil of Submission: Creates a sigil of submission.
 	name = "Sigil of Submission"
-	desc = "Places a subtle sigil that will enslave any valid beings standing on it after a time."
+	desc = "Places a luminous sigil that will enslave any valid beings standing on it after a time."
```
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_scripture.dm]: not flavortext tm

**Comment by coiax** [comment]
And we're sure the success rate is still 100% when overlapping? Unless the person has stun reducing reagents or changeling adrenal powers?

**Comment by ChangelingRain** [comment]
Yes I'm sure, I tested it like five times so it should be either 100% or high enough that the one edge case guy won't resist and be noticed.

**Comment by ChangelingRain** [comment]
They're literally already that easily destroyable, you just need to use harm intent.

**Comment by Razharas** [comment]
Ok i believe joan of this

**Pull request merged**